src/tito/tagger/main.py

```diff
@@ -405,7 +405,7 @@ def _bump_version(self, release=False, zstream=False, force=False):
                                         increase_zstream(match.group(2)),
                                         "\n"
                         ))
-                elif force:
+                elif self._use_version is not None:
```

Comment:
> it looks like if they kept doing this it will now just call their normal configured tagger by default,

Yes

> which probably ignores the version specified

No, the line of code that I am commenting on in this thread is what makes the original `VersionTagger` keep the force behavior.


I will rework this PR to have a no-op `ForceVersionTagger` so we don't break anyone who is implementing their own.